src/routes/docs/features/cloud-drives/+page.md

```diff
@@ -37,54 +37,50 @@ https://www.dropbox.com/install
 
 https://mega.io/desktop
 
-## Amazon Drive
```

Comment:
Is Amazon Drive no longer supported? If so, we should remove from the main app as well.

Reply:
It doesn't exist anymore.